src/config.rs

```diff
@@ -235,8 +233,6 @@ impl Config {
             self.cfg_test = false;
             self.rustfmt_path = None;
             self.build_command = None;
-        } else if self.build_command.is_some() {
-            self.build_on_save = true;
```

Comment:
Why remove this?

Reply:
In the case of `build_command` if we successfully recreate a plan from the save-analysis files, we use that to get a live-edit experience just like in the Cargo case; that's the point of the change.

Do you think we should differentiate between 'always run external build command and on save' vs 'run this external build command as a bootstrap/rebuild command, from which we get a plan we reuse'?

Reply:
That makes sense, thanks. I don't think we need to make that distinction - the `build_on_save` option is basically that.